.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,57 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+---
+name: Problem Report
+about: Create a Report to help us improve
+---
+
+<!-- Thanks for reporting a problem for this project. READ THIS FIRST:
+
+This issue template is meant to REPORT Labrador software PROBLEMS ONLY
+
+Please DO NOT OPEN AN ISSUE:
+  - If your Labrador version is not the latest from the master branch, please update your device before submitting your issue. Your problem might already be solved. 
+The latest pre-release binaries of Labrador can be downloaded from GitHub Actions.
+
+Please take a few minutes to complete the requested information below. Our ability to provide assistance is greatly hampered without it. The details requested potentially affect which options to pursue. The small amount of time you spend completing the template will also help the volunteers providing the assistance to you to reduce the time required to help you.
+
+DO NOT DELETE ANY TEXT from this template! Otherwise the issue will be auto-closed.
+-->
+
+### PROBLEM DESCRIPTION
+_A clear and concise description of what the problem is._
+
+
+### REQUESTED INFORMATION
+_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_
+
+
+- [ ] Searched the problem in [issues](https://github.com/espotek-org/Labrador/issues)
+- [ ] Searched the problem in [discussions](https://github.com/espotek-org/Labrador/discussions)
+- [ ] Searched the problem in the [docs](https://github.com/espotek-org/Labrador/wiki)
```

Comment:
Same with this section here.  Most people will Google the problem before opening a GitHub Issue, and if they can't find it we can simply just redirect them to the already existent thread.

Reply:
> Most people will Google the problem before opening a GitHub Issue,

This section from my view just makes it **absolutely clear to everyone**  what maintainers expect from an issue author as obligation. 
It avoids burning out maintainers therefore I'd like to keep it, maybe with softer wording.

ok?

Reply:
@EspoTek may I kindly remind about my open questions in the review comments?
This PR is stuck without answers to reach a conclusion.

Reply:
Sorry about this, slipped my mind.  Happy to listen to you in this regard.